[2023-10-29T01:06:02.046+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=col_api_etl/run_id=manual__2023-10-29T01:05:58.103122+00:00/task_id=extract_currency_conversion_rates permission to 509
[2023-10-29T01:06:02.047+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=col_api_etl/run_id=manual__2023-10-29T01:05:58.103122+00:00/task_id=extract_currency_conversion_rates permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=col_api_etl/run_id=manual__2023-10-29T01:05:58.103122+00:00/task_id=extract_currency_conversion_rates'
[2023-10-29T01:06:02.075+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=col_api_etl/run_id=manual__2023-10-29T01:05:58.103122+00:00/task_id=extract_currency_conversion_rates permission to 509
[2023-10-29T01:06:02.076+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=col_api_etl/run_id=manual__2023-10-29T01:05:58.103122+00:00/task_id=extract_currency_conversion_rates permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=col_api_etl/run_id=manual__2023-10-29T01:05:58.103122+00:00/task_id=extract_currency_conversion_rates'
[2023-10-29T01:06:02.108+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: col_api_etl.extract_currency_conversion_rates manual__2023-10-29T01:05:58.103122+00:00 [queued]>
[2023-10-29T01:06:02.119+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: col_api_etl.extract_currency_conversion_rates manual__2023-10-29T01:05:58.103122+00:00 [queued]>
[2023-10-29T01:06:02.120+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-10-29T01:06:02.133+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): extract_currency_conversion_rates> on 2023-10-29 01:05:58.103122+00:00
[2023-10-29T01:06:02.139+0000] {standard_task_runner.py:57} INFO - Started process 4100 to run task
[2023-10-29T01:06:02.139+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2023-10-29T01:06:02.144+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'col_api_etl', 'extract_currency_conversion_rates', 'manual__2023-10-29T01:05:58.103122+00:00', '--job-id', '1091', '--raw', '--subdir', 'DAGS_FOLDER/col_api_etl.py', '--cfg-path', '/tmp/tmp5puyqfxu']
[2023-10-29T01:06:02.146+0000] {standard_task_runner.py:85} INFO - Job 1091: Subtask extract_currency_conversion_rates
[2023-10-29T01:06:02.188+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=col_api_etl/run_id=manual__2023-10-29T01:05:58.103122+00:00/task_id=extract_currency_conversion_rates permission to 509
[2023-10-29T01:06:02.188+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=col_api_etl/run_id=manual__2023-10-29T01:05:58.103122+00:00/task_id=extract_currency_conversion_rates permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=col_api_etl/run_id=manual__2023-10-29T01:05:58.103122+00:00/task_id=extract_currency_conversion_rates'
[2023-10-29T01:06:02.191+0000] {task_command.py:415} INFO - Running <TaskInstance: col_api_etl.extract_currency_conversion_rates manual__2023-10-29T01:05:58.103122+00:00 [running]> on host ea85de3a6109
[2023-10-29T01:06:02.267+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='col_api_etl' AIRFLOW_CTX_TASK_ID='extract_currency_conversion_rates' AIRFLOW_CTX_EXECUTION_DATE='2023-10-29T01:05:58.103122+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-29T01:05:58.103122+00:00'
[2023-10-29T01:06:06.041+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-10-29T01:06:06.052+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=col_api_etl, task_id=extract_currency_conversion_rates, execution_date=20231029T010558, start_date=20231029T010602, end_date=20231029T010606
[2023-10-29T01:06:06.087+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
